docs/src/knownissues.md

```diff
@@ -65,8 +65,51 @@ ENV["UCX_ERROR_SIGNALS"] = "SIGILL,SIGBUS,SIGFPE"
 ```
 at `__init__`. If set externally, it should be modified to exclude `SIGSEGV` from the list.
 
+## CUDA-aware MPI
+
+### Memory pool
+
+Using CUDA-aware MPI on multi-GPU nodes with recent CUDA.jl may trigger (see [here](https://github.com/JuliaGPU/CUDA.jl/issues/1053#issue-946826096))
+```
+The call to cuIpcGetMemHandle failed. This means the GPU RDMA protocol
+cannot be used.
+  cuIpcGetMemHandle return value:   1
+```
+in the MPI layer, or fail on a segmentation fault (see [here](https://discourse.julialang.org/t/cuda-aware-mpi-works-on-system-but-not-for-julia/75060?)) with
+```
+[1642930332.032032] [gcn19:4087661:0] gdr_copy_md.c:122 UCX ERROR gdr_pin_buffer failed. length :65536 ret:22
+```
+Reason for this to happen is that the legacy cuIpc* APIs are incompatible with stream-ordered allocator, now default in CUDA.jl.
+
+To circumvent this, one has to ensure the CUDA memory pool to be set to `none`:
+```
+export JULIA_CUDA_MEMORY_POOL=none
+```
+_More about CUDA.jl [memory environment-variables](https://juliagpu.gitlab.io/CUDA.jl/usage/memory/#Environment-variables)._
```

Comment:
Is this still supported? This was removed from the latest docs (https://cuda.juliagpu.org/stable/search/?q=JULIA_CUDA_MEMORY_POOL) and @vchuravy's comment suggests it may no longer work https://github.com/JuliaGPU/CUDA.jl/issues/1053#issuecomment-881914583?

Reply:
Interesting, thanks for flagging this @simonbyrne. Environment variable section indeed no longer exists in latest CUDA.jl doc on memory pools (https://cuda.juliagpu.org/v3.7/usage/memory/#Memory-pool).

Testing on a 8x Tesla A100 system with latest cuda drivers stack and Julia CUDA v3.7.0 returns following when running the CUDA-aware MPI [test MWE](https://discourse.julialang.org/t/cuda-aware-mpi-works-on-system-but-not-for-julia/75060/11): 
- without tweeting any of the memory pool variables
```
julia> CUDA.memory_status()
Effective GPU memory usage: 1.02% (414.000 MiB/39.586 GiB)
Memory pool usage: 0 bytes (0 bytes reserved)
# [...]
mpirun -np 8 julia mpi_test.jl
# [...]
The call to cuIpcGetMemHandle failed. This means the GPU RDMA protocol
cannot be used.
  cuIpcGetMemHandle return value:   1
# [...]
71 more processes have sent help message help-mpi-common-cuda.txt / cuIpcGetMemHandle failed
```

- with explicitly setting `export JULIA_CUDA_MEMORY_POOL=none`
```
export JULIA_CUDA_MEMORY_POOL=none

julia> CUDA.memory_status()
Effective GPU memory usage: 1.02% (414.000 MiB/39.586 GiB)
No memory pool is in use.
# [...]
mpirun -np 8 julia mpi_test.jl
success
```

Any comments on that @vchuravy and @maleadt ?

Reply:
Documentation of the env var had been removed because the alternative pools (split, binned, etc) are gone. The only possible options now are `cuda` and `none`, and I was hoping to get rid of the env var altogether, but apparently it's needed for use with the legacy IPC mechanisms as used by some HPC libraries. So we should probably add back some minimal documentation.

Reply:
I guess that's be good to have the minimal doc about `cuda` and `none` pool indeed, as it is still needed (on all 3 systems I tried).

Reply:
okay, i'll leave this link until the CUDA.jl docs are updated.

Reply:
It's an old link (gitlab.juliagpu.io) that may disappear any time, so it's better to update that already. I'll add a bit to https://cuda.juliagpu.org/v3.7/usage/memory/#Memory-pool.

Reply:
https://github.com/JuliaGPU/CUDA.jl/commit/18dc2367e69d19a58b27c8f6068c6753f4c51dd9

Reply:
Thanks for adding some infos to the CUDA doc @maleadt! I guess one should still correct the link @simonbyrne ?

Reply:
I've updated think link on master, thanks!